docs/user-guide/projects/README.md

```diff
@@ -201,32 +201,29 @@ To get an overview of all your projects, select the **View all projects** option
 
 ![view all projects](1566293251039.png)
 
-You will then get a list of all your projects in OpenProject.
+You will then get a list of all your projects in OpenProject. You can use this projects overview to **create a multi project status dashboard** if you include your own [project custom fields](../../system-admin-guide/custom-fields/custom-fields-projects/), e.g. custom status options, Accountable, Project duration, and more.
 
-With the **three dots** icon at the right in the list you can open further features, such as [creating a new subproject](#create-a-subproject), [project settings](project-settings), [archiving a project](#archive-a-project), [copying](#copy-a-project) and [deleting a project](#delete-a-project).
+With the horizontal **three dots** icons at the right in the list you can open **further features**, such as [creating a new subproject](#create-a-subproject), [project settings](project-settings), [archiving a project](#archive-a-project), [copying](#copy-a-project) and [deleting a project](#delete-a-project). With the **arrow** you can display the **project description**.
 
```

Comment:
We could add a comment that a user needs to be a system admin to access these options.